Add Rise AI Source documentation (#8105)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/rise-ai/index.md

4. Click **Add Source** to save your settings.
5. Copy the Write key from the Segment UI.
6. Log in to your Rise AI account - navigate to Settings > Integrations > Segment Integration and paste the key to connect.

## Stream

Rise AI uses our stream Source component to send Segment event data. It uses a server-side `track` method to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL.

Rise AI includes the `userId` when available, along with a unique `tenant_id` for tenant isolation to ensure data privacy and proper attribution across different workspaces.

## Events

The table below lists events that Rise AI sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations. Rise AI includes the `userId` if available.

> | Event Name          | Description                                                        |

Reviewer: ```suggestion
| Event name          | Description                                                        |
```

| ------------------- | ------------------------------------------------------------------ |
| walkthrough-progress | User progress through AI-guided walkthroughs and onboarding flows |
| chats               | AI chat session creation and interactions                          |

## Event Properties

The table below list the properties included in the events listed above.

| Property Name             | Description                                                      |
| ------------------------- | ---------------------------------------------------------------- |
| `batch_id`                | Unique identifier for the event batch                            |
| `chat_id`                 | Unique identifier for the chat session                           |

> | `environment`             | Environment where the event occurred (production, staging, etc.) |

Reviewer: ```suggestion
| `environment`             | Environment where the event occurred (for example, production or staging) |
```

| `event_timestamp`         | Timestamp when the event occurred                                |
| `tenant_id`               | Unique identifier for the tenant/workspace                       |
| `step_index`              | Position of the current step in the walkthrough (walkthrough-progress only) |
| `step_title`              | Title of the current walkthrough step (walkthrough-progress only) |
| `fulfillment_action`      | Action taken to fulfill the step (walkthrough-progress only)     |
| `walkthrough_job_to_be_done` | The job-to-be-done that the walkthrough helps accomplish (walkthrough-progress only) |
| `chat_type`               | Type of chat interaction (chats only)                            |

## Adding Destinations

> Now that your Source is set up, you can connect it with Destinations.

Reviewer: ```suggestion
Now that your source is set up, you can connect it with destinations.
```

> Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don't appear, check the [Event Delivery](/docs/connections/event-delivery/) tool, and refer to the Destination docs for each tool for troubleshooting.

Reviewer: ```suggestion
Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don't appear, check the [Event Delivery](/docs/connections/event-delivery/) tool, and refer to the destination docs for each tool for troubleshooting.
```
